### PR TITLE
Add Voice Over support on watchlist Article and Diff actions

### PR DESCRIPTION
**Pull request by tonisevener**
**Phabricator:**
https://phabricator.wikimedia.org/T335816

### Notes
This adds some Voice Over handling on the article and diff watch / unwatch buttons. To avoid merge conflicts this is stacked onto #4596.

### Test Steps
1. Enable Voice Over, select Watch on the Article or Diff view.
2. Confirm Watch success message is announced. Confirm user can navigate to "View Watchlist" button and activate it.
3. Repeat step 1 for unwatch. Confirm unwatch message is announced.
4. Repeat step 1 on airplane mode. Confirm "No Internet Connection" message is announced.

### CODE REVIEW

**Review by staykids — APPROVED** [review]

**Comment by staykids** [comment]
Apologies – I'm so used to hitting merge after approval I mindlessly hit it without considering the dependent PR label. I opened a PR to revert my merge (https://github.com/wikimedia/wikipedia-ios/pull/4609), then I'll re-open this with same branch, approve it, and wait for the dependent PR to get merged before merging. Sorry again.

**Comment by tonisevener** [comment]
@staykids No problem! Thanks for jumping on it.

**Pull request merged**
